#56 New sideMenu

Sky-De opened the pull request:
# Issue #56📐

## DONE
With minimum of code change i implemented minimal design but match to design of project with nice UI , also it has smooth transition while toggling show/hidden 

## CONSIDER
merge second commit / the first one is not finished 

## Guidelines 🔐

I accept the fact that I have followed the guidelines and have not copied the codes from around the internet

- [x] **Contribution Guidelines**
- [x] **Code of Conduct**

## Issue to be closed 🛅

- My pull request closes #56

## Screenshots 📷


Here are the pictures of changes that I have made 🔽

**DARK MODE**

![darkSide](https://github.com/warmachine028/nike/assets/79264045/a9c6495f-f8f4-40e4-b47f-b10546c5dd3c)



**LIGHT MODE**

![lightSide](https://github.com/warmachine028/nike/assets/79264045/43fbd9f0-34d5-4f28-a9d7-4016abe564b6)

  
<!-- Please refer to previous closed issues for better understanding of filling this template -->  
  
---

warmachine028 reviewed (changes requested):
![263094181-5e3fd063-0383-4554-ad48-d7a966872f9b](https://github.com/warmachine028/nike/assets/75939390/423734d4-4ac5-4fe4-907d-22714c9294b8)

- The blur effects must be added in the side of the menu

Sky-De commented:
@warmachine028 hi again, i refactored menu based on design which you provided, 
now it is match to design and also has functionality to click on blur area and it will be closed
also i added jordan svg from original nike website
i hope it be exactly what you want


## Screenshots 📷

Here are the pictures of changes that I have made 🔽
![menuDark](https://github.com/warmachine028/nike/assets/79264045/b3dd6ecb-9e2e-4d3a-9be6-bea6b2c572cd)
![menuLight](https://github.com/warmachine028/nike/assets/79264045/1fff295f-d2b6-430b-bb1b-da6e54ee5be2)

if you got any question feel free to ask, otherwise merge this then i will ask for assignment for an another issue
thanks

warmachine028 reviewed (approved):
### Fantastic 🎉

Looks good now
